src/components/Hero/Hero.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react'
+import './Hero.scss'
+import { useDispatch } from 'react-redux';
+import { DECREMENT, INCREMENT } from '../../store/actions/types';
+import { decreamentAction, increamentAction } from '../../store/actions/cartActions';
+
+const Hero = (props) => {
+  const content = props.content;
+
+  const dispatch = useDispatch();
+
+  const addToCart = ()=> {
+    increamentAction(dispatch)
+  }
+
+  const removeFromCart = ()=> {
+    decreamentAction(dispatch)
+  }
+
+  return (
+    <div className='hero-container'>
+      <div className='hero__context'>
+        <span className='hero__context-sub-title'>
+          {content && content.subtitle || 'Plants'}
```

Comment:
In this case it would be better to add it as `{content.subtitle}` and add a default subtitle to the content  by default instead of doing it here. 

Also reducing the amount of conditions in render method us very handy for more readability as well for more important thing is awareness of amount of re-rendering.

In case of adding conditions in render it's always better to add them as separate variables if possible